### PR TITLE
IS-66 - Create the Virtual Classrooms program view

### PR DESCRIPTION
**Pull request by crayolakat**

### CODE REVIEW

**Comment by bibekg** [comment]
Glad to see progress! Make sure you change the base branch on this PR to `programs-page-redesign` since we want to aggregate these changes there first. 

Also, did you create this branch off of `master` or that branch?

**Comment by crayolakat** [comment]
I created this branch off the programs-page-redesign branch

**Comment by bibekg** [comment]
Okay cool, in that case, just change the base branch for the PR, as described above.

**Comment by crayolakat** [comment]
Okay, I've changed the base.

**Comment by bibekg** [comment]
Just took a look at it in-browser. Looks great! Only thing I'd suggest changing up is reducing the upper/lower margin on the text and adding a little bit of padding on the sides to make it more closely match with the mockup.
image

**Comment by crayolakat** [comment]
I added padding to the sides and increased the line height to reduce the upper/lower margins. I think the upper/lower margins still look a bit too big, do you think I should increase the text size or crop the picture?

**Comment by bibekg** [comment]
I think you should decrease the text size a little bit (maybe to `$text_sm` or `$text_md`) and then reduce your `$program_footer_height` SCSS variable to about 500px.

**Comment by crayolakat** [comment]
Made the changes, what do you think now?

**Comment by crayolakat** [comment]
I also added a few things to the code to make it compatible with Internet Explorer.

File: src/views/Navbar/NavbarDropdown/NavbarDropdown.js [hunk 1]
```diff
@@ -57,6 +57,50 @@ export default function NavbarDropdown(props) {
 
   // Looks into the 'sections' property exported by each page component to see
   // what anchor-links that page makes available
+  //Polyfill for Array.prototype.find (method is not compatible with Internet Explorer)
```
Comment (bibekg) [inline src/views/Navbar/NavbarDropdown/NavbarDropdown.js]: Nice, great work finding the IE bug!

Only comment here is, polyfills should be not be placed within a React component. A good idea would be to move it to `src/utilities/polyfill.js`, then just throw in a 
```js
import './utilities/polyfill'
```
at the top of `src/index.js`

**Review by bibekg — CHANGES_REQUESTED** [review]
Left a comment on a the polyfill. Also, I'd like to suggest a few other changes:
1. Move the ProgramFooter component from `components` to `pages/OurProgramsPage/ProgramViews` since it's not really going to be used anywhere besides ProgramViews.
2. Same thing goes for the `views/VirutalClassroom` component. It should just be located within `pages/OurProgramsPage/VirtualClassroomsView`
3. I noticed there are two instances of the footer image in this pull request as both `footer.jpg` and `virtual-learning.jpg`. Delete one and resize the other so it has a width of ~1000px in order to reduce file size/page load time.

**Comment by crayolakat** [comment]
I made the changes and fixed another IE bug that I noticed. I actually noticed a lot of IE bugs, but have not had the time to fix. Do you want me to start logging them somewhere?

**Comment by bibekg** [comment]
That would be great. Just file them as issues here on the Github.

**Review by bibekg — APPROVED** [review]

**Comment by bibekg** [comment]
Good work. 👍 I've assigned the Learning Disabilities program view to you to work on next.

**Pull request merged**